examples/community-detection.ipynb

```diff
@@ -0,0 +1,1148 @@
+       "      <td>2579</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>[playmygame, circlebroke, tribes, conspiratard...</td>\n",
+       "      <td>2676</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>[radioreddit, autism, modhelp, digital_immorta...</td>\n",
+       "      <td>3158</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>295</th>\n",
+       "      <td>[banishedmaps]</td>\n",
+       "      <td>3032</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>296</th>\n",
+       "      <td>[screenshots]</td>\n",
+       "      <td>3034</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>297</th>\n",
+       "      <td>[leangains]</td>\n",
+       "      <td>3039</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>298</th>\n",
+       "      <td>[agnostic]</td>\n",
+       "      <td>3040</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>299</th>\n",
+       "      <td>[mario]</td>\n",
+       "      <td>3043</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>300 rows × 2 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                            Subreddits  communityId\n",
+       "0    [airsoft, bandnames, connecticut, thehiddenbar...         2406\n",
+       "1    [posthardcore, metalcore, corejerk, iama, karm...         2612\n",
+       "2    [locationbot, oldschoolcoolnsfw, uncomfortable...         2579\n",
+       "3    [playmygame, circlebroke, tribes, conspiratard...         2676\n",
+       "4    [radioreddit, autism, modhelp, digital_immorta...         3158\n",
+       "..                                                 ...          ...\n",
+       "295                                     [banishedmaps]         3032\n",
+       "296                                      [screenshots]         3034\n",
+       "297                                        [leangains]         3039\n",
+       "298                                         [agnostic]         3040\n",
+       "299                                            [mario]         3043\n",
+       "\n",
+       "[300 rows x 2 columns]"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "query = \"\"\"\n",
+    "    CALL gds.louvain.stream('largest_connected_components2')\n",
+    "    YIELD nodeId, communityId, intermediateCommunityIds\n",
+    "    RETURN collect(gds.util.asNode(nodeId).node_id) AS Subreddits, communityId\n",
+    "    ORDER BY size(Subreddits) DESC\n",
+    "\"\"\"\n",
+    "wcc = gds.run_cypher(query)\n",
+    "wcc"
+   ]
+  },
+  {
```

Comment:
Did you draw some conclusions about the created communities?

Maybe we could also add a section about visualizing the community.
I would suggest to just link to Bloom which users can follow up on if they like